### PR TITLE
Move some dist/ files into src/

### PR DESCRIPTION
**Pull request by avsingh999**
#120 fixed
@jywarren @publiclab/mentors  @sagarpreet-chadha @gauravano  please review

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {
   "name": "leaflet-environmental-layers",
```
Comment (jywarren) [inline package-lock.json]: Some of the changes in this file are downgrades... Would you mind removing it from this PR? Thanks!

Comment (avsingh999) [reply package-lock.json]: Sorry I forgot to remove this file

**Comment by avsingh999** [comment]
@jywarren I removed that fil. No please review again.
thanks :)

**Comment by jywarren** [comment]
Ah, sorry - can you add back in the same version that's currently our `main` branch? We don't want to delete it, just not change it. Thanks!!!

**Comment by avsingh999** [comment]
@jywarren 

> Ah, sorry - can you add back in the same version that's currently our `main` branch? We don't want to delete it, just not change it. Thanks!!!

@jywarren  no problem I will do but I want to know why lock file use it's automatically generated while npm install?
thanks : )

**Comment by avsingh999** [comment]
@jywarren  I add package-lock.json file

**Comment by jywarren** [comment]
Great! Thanks so much!!!

**Pull request merged**